packages/webapp/pages/onboarding.tsx

```diff
@@ -368,15 +396,16 @@ export function OnboardPage(): ReactElement {
   const shouldShowOnline = useFeature(feature.onboardingOnlineUsers);
 
   const getProgressBar = () => {
-    if (isFiltering) {
+    if (activeScreen !== OnboardingStep.Intro) {
       return null;
     }
 
-    const percentage = isFiltering ? 100 : 50;
+    const percentage = 50;
     return <ProgressBar percentage={isAuthenticating ? percentage : 0} />;
```

Comment:
So now is 0 or 50 not 50 or 100?

Reply:
They above always said:
```
if (isFiltering) {
      return null;
    }
```
    
So would never even come here 😅
Probably legacy experiment stuff.

Reply:
😅